cmd/gazelle/diff.go

```diff
@@ -36,7 +36,10 @@ func diffFile(c *config.Config, f *rule.File) error {
 	}
 	rel = filepath.ToSlash(rel)
 
-	date := "1970-01-01 00:00:00.000000000 +0000"
+	// Use a dummy timestamp that is not the epoch, so use one ns past the epoch.
```

Comment:
Could you mention here that we do this since the epoch is treated specially by some tools?